configs/rover_configs.yaml

```diff
@@ -32,4 +32,28 @@ core:
     # How frequently to check the connection(second)
     check_period: ${ROVER_BACKEND_CHECK_PERIOD:5}
     # The auth value when send request
-    authentication: ${ROVER_BACKEND_AUTHENTICATION:""}
\ No newline at end of file
+    authentication: ${ROVER_BACKEND_AUTHENTICATION:""}
+
+process_discovery:
+  active: ${ROVER_PROCESS_DISCOVERY_ACTIVE:true}
+  # The period of report or keep alive process(second)
+  heartbeat_period: ${ROVER_PROCESS_DISCOVERY_HEARTBEAT_PERIOD:20}
+  # Detect processes in VM mode
+  vm:
+    # Is active the VM mode to detect processes
+    active: ${ROVER_PROCESS_DISCOVERY_VM_ACTIVE:true}
+    # The period to detect the process(second)
+    period: ${ROVER_PROCESS_DISCOVERY_VM_PERIOD:15}
```

Comment:
This is a local mechanism, let's make it more accurate. Try ms as unit, and set 500ms as default.

Reply:
Updated to use the time duration format, and make the default is `3s`, 500ms may too frequently.